From 4a1db2699d1bec63d7124a3d49ac2597b2852f87 Mon Sep 17 00:00:00 2001
From: Markus Armbruster <armbru@redhat.com>
Date: Mon, 11 Aug 2014 07:54:48 +0200
Subject: [PATCH 14/16] rhel: *.power_controller_present=off for machine types <= 7.0

Message-id: <1407743689-13553-15-git-send-email-armbru@redhat.com>
Patchwork-id: 60507
O-Subject: [PATCH RHEV-7.1 qemu-kvm-rhev 14/15] rhel: *.power_controller_present=off for machine types <= 7.0
Bugzilla: 1118665
RH-Acked-by: Eduardo Habkost <ehabkost@redhat.com>
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>
RH-Acked-by: Dr. David Alan Gilbert (git) <dgilbert@redhat.com>

Update RHEL compatibility properties for upstream commit f23b6bd.

Adding to PC_RHEL7_0_COMPAT even though only q35 types need it, just
to stay close to upstream, which has it in PC_COMPAT_2_0.

Signed-off-by: Markus Armbruster <armbru@redhat.com>
---
 include/hw/i386/pc.h | 8 ++++++++
 1 file changed, 8 insertions(+)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 include/hw/i386/pc.h |    8 ++++++++
 1 files changed, 8 insertions(+), 0 deletions(-)

diff --git a/include/hw/i386/pc.h b/include/hw/i386/pc.h
index de0570c..0c701e8 100644
--- a/include/hw/i386/pc.h
+++ b/include/hw/i386/pc.h
@@ -553,6 +553,14 @@ bool e820_get_entry(int, uint32_t, uint64_t *, uint64_t *);
             .property = "memory-hotplug-support",\
             .value    = "off",\
         },{\
+            .driver   = "xio3130-downstream",\
+            .property = COMPAT_PROP_PCP,\
+            .value    = "off",\
+        },{\
+            .driver   = "ioh3420",\
+            .property = COMPAT_PROP_PCP,\
+            .value    = "off",\
+        },{\
             .driver   = "PIIX4_PM",\
             .property = "acpi-pci-hotplug-with-bridge-support",\
             .value    = "off",\
-- 
1.7.1

